tools/cli/commands.go

```diff
@@ -383,13 +390,35 @@ func showHistoryHelper(c *cli.Context, wid, rid string) {
 	table.SetBorder(false)
 	table.SetColumnSeparator("")
 	for _, e := range history.Events {
+		columns := []string{}
+		if printFully {
+			jsonBs, err := json.Marshal(e)
```

Comment:
Let's also recursively decode the internal byte[] like "HistoryEventToString" did